pkg/asset/manifests/powervs/ovnkubeconfig.go

```diff
@@ -0,0 +1,47 @@
+package powervs
```

Comment:
This looks generic--not powervs specific. If that is true, could we move this to a more generic package or into network.go could be fine too

Reply:
@patrickdillon i moved this over into just network.go. right now, the only thing configurable is what we want. i'm envisioning that as others need to tweak other params it can be built upon. lmk what you think